packages/apollos-church-api/src/data/content-items/data-source.js

```diff
@@ -203,4 +203,13 @@ export default class ContentItem extends oldContentItem.dataSource {
     });
     return features;
   }
+
+  getBySlug = async (slug) => {
+    const contentItem = await this.request('ContentChannelItemSlugs')
```

Comment:
```suggestion
    const contentItemSlug = await this.request('ContentChannelItemSlugs')
```

Reply:
Should this just be the original contentItemFromSlug? haha

Reply:
it's not a content item table, it's a slug table